.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
```

Comment:
I usually put the linters matrix into a separate file, this allows having a separate Checks section, not mixing up with the actual tests.

Reply:
Could you elaborate on that? What's the benefit?

Reply:
https://github.com/ansible/ansible-lint/runs/638612156:
<img width="403" alt="Screen Shot 2020-05-02 at 10 47 24 PM" src="https://user-images.githubusercontent.com/578543/80891860-12cc9780-8cc7-11ea-96ea-7279bb778681.png">

Reply:
When there's ~100 jobs, it's way easier when at least part of them are filtered out

Reply:
Makes sense, let's move to a separate workflow then.

Reply:
Also, for better or for worse, separate workflow failures won't prevent tests from running, which is rather good IMHO.

Reply:
That depends on the POV, but I think it's for good too.

Reply:
Split up into ci.yml and qa.yaml in 988dfbc.